charts/rancher-gke-operator/1.0.100/values.yaml

```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+gkeOperator:
+  image:
+    repository: rancher/gke-operator
+    tag: v1.0.1-rc12
```

Comment:
This will need to be real release before we merge

Reply:
cc @cmurphy

Reply:
To be clear its correct for now. GKE still has open issues. We just need to remind ourselves to cut one later.

Reply:
Is this good with `v1.0.1-rc13` or are changing it still?